bitnami/mongodb/templates/backup/pvc.yaml

```
@@ -0,0 +1,41 @@
+{{- /*
+Copyright VMware, Inc.
+SPDX-License-Identifier: APACHE-2.0
+*/}}
+
+{{- if and .Values.backup.enabled (not .Values.backup.cronjob.storage.existingClaim) -}}
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: {{ include "mongodb.fullname" . }}-mongodump
+  namespace: {{ include "mongodb.namespace" . | quote }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    app.kubernetes.io/component: mongodump
+    {{- if .Values.backup.cronjob.labels }}
+    {{- include "common.tplvalues.render" (dict "value" .Values.backup.cronjob.labels "context" $) | nindent 4 }}
+    {{- end }}
+    {{- if .Values.commonLabels }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
+    {{- end }}
+  {{- if or .Values.backup.cronjob.annotations .Values.commonAnnotations .Values.backup.cronjob.storage.resourcePolicy}}
+  annotations:
+    {{- if .Values.backup.cronjob.annotations }}
```

This should be `.Values.backup.cronjob.storage.annotations` right?

---

I've got it at the same level as `schedule`.  I could move it under `storage` if you think that would make more sense.

---

I'm not sure I understood you. In my previous comment I was asking because you added `.Values.backup.cronjob.annotations` to the cronjob template (that's fine) but also here, in the pvc template. Unless I'm mistaken, you are not using `.Values.backup.cronjob.storage.annotations` anywhere in the templates so this code should be:

```suggestion
    {{- if .Values.backup.cronjob.storage.annotations }}
```